src/transformers/models/idefics3/configuration_idefics3.py

```diff
@@ -197,7 +197,7 @@ def __init__(
             self.vision_config = vision_config
 
         if isinstance(text_config, dict):
-            text_config["model_type"] = text_config["model_type"] if "model_type" in text_config else "llama"
+            text_config["model_type"] = "llama"
```

Comment:
Why remove this? Technically the model can load any language model in whereas this hard-codes to llama

Reply:
You're right, I re-added it. I thought this might be why merve was running into an error but it was a mix-up with the cache on her machine.